packages/web/src/pages/hustlers/[id]/customize.tsx

```diff
@@ -165,7 +165,16 @@ const Nav = () => (
   </AppWindowNavBar>
 );
 
+type Router = {
+  query: {
+    id: string;
+  };
+};
+
 const Hustlers = () => {
+  const router = useRouter() as unknown as Router;
```

Comment:
@smakosh not sure if there is a better way to do this

Reply:
```ts
const router = useRouter()

const { data, isFetching: loading } = useHustlerQuery({
    where: {
      id: String(router.query.id)
    },
  }, {
    enabled: !!router.query.id
  });

Reply:
Hey @smakosh I've just added this change as discussed with @tarrencev